Added the review for the Che 7 End-user guide once again. (#783)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc

For the purpose of providing tool isolation and easy extensibility in Eclipse Che, the Che-Theia IDE has a set of plug-in APIs. The APIs are compatible with Visual Studio Code extension APIs. In most cases, Che-Theia can run VS Code extensions as its own plug-ins.
// TODO: VS Code extensions as Che-Theia plug-ins. Read more at link:[LINK].

> If the plug-in that you are authoring depends on or interacts with components of Che workspaces (containers, preferences, factories), you can rely on the Che APIs that are embedded in Che-Theia.

Reviewer: ```suggestion
When developing a plug-in that depends on or interacts with components of Che workspaces (containers, preferences, factories), use the Che APIs embedded in Che-Theia.
```
@supriya-bharadwaj, no "if", no "you", please.

A Che-Theia plug-in can be based on a VS Code extension or an Eclipse Theia plug-in.

> A Visual Studio Code extension:: If you have already built your plug-in as a Visual Studio Code extension, you can build the corresponding Che-Theia plug-in by repackaging the extension. If the extension relies on its own set of dependencies, it should package those into a container. Eclipse Che users get the plug-in and do not have to worry about the tool dependencies. See xref:converting-a-visual-studio-code-extension-to-a-che-theia-plug-in_developing-che-theia-plug-ins[].

Reviewer: ```suggestion
A Visual Studio Code extension:: To repackage a VS Code extension as a Che-Theia plug-in with its own set of dependencies, package the dependencies into a container. This ensures that Eclipse Che users do not need to install the dependencies when using the extension. See link:{{site.baseurl}}che-7/using-a-visual-studio-code-extension-in-che[Using a Visual Studio Code extension in Che].
```
@supriya-bharadwaj,
* no 'if', no 'you', no 'should'
* links need to preserved from master and tested (no xrefs)
